### PR TITLE
Add function to add a single headline from my tasks

### PR DESCRIPTION
**Pull request by ration**
Add function to create single story element from a prompted list of active stories.

### CODE REVIEW

**Comment by ration** [comment]
I seem to be use `org-clubhouse` like this more and more i.e. just pick one story from my TODO list in clubhouse, add a org element from it and add my notes to it.

I'm less confident that I followed all the architectural styles on this one so if it needs some work let me know.

File: org-clubhouse.el [hunk 1]
```diff
@@ -1030,6 +1030,16 @@ which labels to set."
...
        ""))))
 
+(defun org-clubhouse-headline-from-my-tasks (level)
+  "Prompt my active stories and create a single `org-mode' headline at LEVEL."
+  (interactive "*nLevel: \n")
+    (if org-clubhouse-username
+        (let* ((story-list (org-clubhouse--search-stories (format "owner:%s !is:done !is:archived" org-clubhouse-username)))
+               (stories (to-id-name-pairs story-list)))
+          (org-clubhouse-headline-from-story-id level
+                                                (find-match-in-alist (ivy-read "Select Story: " (-map #'cdr stories)) stories)))
```
Comment (glittershark) [inline org-clubhouse.el:1040]: I usually try to wrap as close to 80 chars as is reasonable

File: org-clubhouse.el [hunk 2]
```diff
@@ -1030,6 +1030,16 @@ which labels to set."
...
+    (warn "Can't fetch my tasks `org-clubhouse-username' is unset")))
```
Comment (glittershark) [inline org-clubhouse.el:1041]: 1. this should probably be `user-error`
2. missing an "if" ("can't fetch my tasks *if*)

Comment (ration) [reply org-clubhouse.el]: I copied the warn from line 1166. If you want in some other way can you show my how. 

I squashed the fix with the rows now shorter.

Comment (glittershark) [reply org-clubhouse.el]: ah, yeah, that's fair. I'll make the change project-wide if I decide to.

**Pull request merged**